pumps/mongo_aggregate.go

```diff
@@ -274,9 +261,7 @@ func (m *MongoAggregatePump) WriteData(ctx context.Context, data []interface{})
 			_, err := analyticsCollection.Find(query).Apply(change, &doc)
 
 			if err != nil {
-				log.WithFields(logrus.Fields{
-					"prefix": analytics.MongoAggregatePrefix,
-				}).Error("UPSERT Failure: ", err)
+				m.log.Error("UPSERT Failure: ", err)
```

Comment:
@tbuchaillot sorry I missed it but can we please add more info here - the query and the change? the query has the org id and also the actual query that failed. I'm speaking from first-hand experience trying to debug an issue between pump and mongo, in production

Reply:
Added query field :)